transformer_engine/paddle/fp8.py

```diff
@@ -235,25 +246,44 @@ def amax_and_scale_update(
     fp8_meta: Dict[str, Any],
     fwd_update: bool,
     update_weight_scale_inv: bool = True,
+    current_step_id_tensor: Optional[Any] = None,
```

Comment:
```suggestion
    current_step_id_tensor: Optional[paddle.Tensor] = None,
```
It's not intuitive to me that this is a CPU `int` tensor, so it may be worth documenting or adding a comment.